app/lib/metadata.py

```diff
@@ -4,10 +4,10 @@
 
 import os
 import yaml
+import traceback
 
-from lib.composegenerator.next.stage1 import createCleanConfigFromV3
 from lib.composegenerator.v2.networking import getMainContainer
-from lib.composegenerator.v1.networking import getFreePort
+from lib.composegenerator.shared.networking import assignIpV4
```

Comment:
I don't see this file anywhere, could it be missing?

Reply:
This should be fixed now